src/components/FlowRunResults.vue

```diff
@@ -3,46 +3,46 @@
     <template v-if="hasResults">
       <div class="flow-run-results__button-group-container">
         <slot name="actions" />
-        <p-button-group v-model="activeViewMode" :options="viewOptions" />
+        <ViewModeButtonGroup class="flow-run-results__view-mode-button-group" />
       </div>
 
       <p-heading heading="6" class="flow-run-results__subheading">
         {{ localization.info.flowRun }}
       </p-heading>
-      <div class="flow-run-results__list" :class="classes.list">
-        <template v-if="flowRunResults.length">
+      <template v-if="flowRunResults.length">
```

Comment:
I noticed all your RowGridLayout's have a v-if checking the array length. Is that necessary? Does it not handle being empty gracefully?

Reply:
Oh it does, these are to make sure we show the empty state if there are no results of that specific lineage (flow run / task run)

Reply:
Then my nit would be let the RowGridLayout do its thing (without the v-if) and add an `empty` computed and v-if that to show the empty state. Keeps our checks truthy and gets rid of a v-else in the template.